### PR TITLE
Bug 1767308 - Re-pin requirements for scripts using mozilla-version 

### PR DESCRIPTION
**Pull request by gbrownmozilla**
Updated with pin.sh.

Changes for esr-102 in mozilla-version 1.0.1 will be needed by treescript, at least.

### CODE REVIEW

File: beetmoverscript/requirements/base.py38.txt [hunk 1]
```diff
@@ -180,27 +180,29 @@ charset-normalizer==2.0.12 \
...
-    --hash=sha256:74d6c7e80609c0f4c2434b97b80c7f8fdfaa072ca4baab7e239a15d6d70ed73a \
-    --hash=sha256:7be0eec337359c155df191d6ae00a5e8bbb63933883f4f5dffc439dac5348c3f \
-    --hash=sha256:94ae132f0e40fe48f310bba63f477f14a43116f05ddb69d6fa31e93f05848ae2 \
-    --hash=sha256:bb5829d027ff82aa872d76158919045a7c1e91fbf241aec32cb07956e9ebd3c9 \
-    --hash=sha256:ca238ceb7ba0bdf6ce88c1b74a87bffcee5afbfa1e41e173b1ceb095b39add46 \
-    --hash=sha256:ca28641954f767f9822c24e927ad894d45d5a1e501767599647259cbf030b903 \
-    --hash=sha256:e0344c14c9cb89e76eb6a060e67980c9e35b3f36691e15e1b7a9e58a0a6c6dc3 \
-    --hash=sha256:ebc15b1c22e55c4d5566e3ca4db8689470a0ca2babef8e3a9ee057a8b82ce4b1 \
-    --hash=sha256:ec63da4e7e4a5f924b90af42eddf20b698a70e58d86a72d943857c4c6045b3ee
+cryptography==37.0.1 \
```
Comment (escapewindow) [inline beetmoverscript/requirements/base.py38.txt:183]: https://cryptography.io/en/latest/changelog/#v37-0-0

Deprecating a number of things. I haven't dug through -- this is probably worth monitoring a bit after rollout, and/or a test in staging. We may want to dig to see what versions of OpenSSL we're using and compare this to the cryptography roadmap -- probably in signingscript.

File: beetmoverscript/requirements/base.py38.txt [hunk 2]
```diff
@@ -288,17 +290,17 @@ immutabledict==2.2.1 \
...
-    --hash=sha256:077ce6014f7b40d03b47d1f1ca4b0fc8328a692bd284016f806ed0eaca390ad8 \
-    --hash=sha256:611bb273cd68f3b993fabdc4064fc858c5b47a973cb5aa7999ec1ba405c87cd7
+jinja2==3.1.2 \
+    --hash=sha256:31351a702a408a9e7595a8fc6150fc3f43bb6bf7e319770cbc0db9df9437e852 \
+    --hash=sha256:6088930bfe239f0e6710546ab9c19c9ef35e29792895fed6e6e31a023a182a61
     # via -r requirements/base.in
-jmespath==0.10.0 \
-    --hash=sha256:b85d0567b8666149a93172712e68920734333c0ce7e89b78b3e987f71e5ed4f9 \
-    --hash=sha256:cdf6525904cc597730141d61b36f2e4b8ecc257c420fa2f4549bac2c2d0cb72f
+jmespath==1.0.0 \
```
Comment (escapewindow) [inline beetmoverscript/requirements/base.py38.txt:301]: Dropping py36 support, we should be good. https://github.com/jmespath/jmespath.py/blob/develop/CHANGELOG.rst#100

File: pushapkscript/requirements/base.py38.txt [hunk 3]
```diff
@@ -899,12 +902,13 @@ uritemplate==4.1.1 \
...
-urllib3==1.26.8 \
-    --hash=sha256:000ca7f471a233c2251c6c7023ee85305721bfdf18621ebff4fd17a8653427ed \
-    --hash=sha256:0e7c33d9a63e7ddfcb86780aac87befc2fbddf46c58dbb487e0855f7ceec283c
+urllib3==1.26.9 \
+    --hash=sha256:44ece4d53fb1706f667c9bd1c648f5469a2ec925fcf3a776667042d645472c14 \
+    --hash=sha256:aabaf16477806a5e1dd19aa41f8c2b7950dd3c746362d7e3223dbe6de6ac448e
     # via requests
-voluptuous==0.12.2 \
-    --hash=sha256:4db1ac5079db9249820d49c891cb4660a6f8cae350491210abce741fabf56513
+voluptuous==0.13.1 \
```
Comment (escapewindow) [inline pushapkscript/requirements/base.py38.txt:909]: ! I thought voluptuous was going to bitrot. Nothing jumping out at me https://github.com/alecthomas/voluptuous/blob/master/CHANGELOG.md

**Review by escapewindow — APPROVED** [review]

File: beetmoverscript/requirements/base.py38.txt (hunk 1 above)
Comment (gbrownmozilla) [reply beetmoverscript/requirements/base.py38.txt:183]: Thanks for pointing that out. I see what you mean. Proceeding with caution...

**Pull request merged**